templates/security/login.html.twig

```diff
@@ -22,13 +22,17 @@
         <div class="login__logo">
             <img class="logo" alt="Bolt CMS logo"
                 {% if not config.get('general/omit_backgrounds') %}
-                 src="{{ asset('assets/images/bolt_logo_loginpage--bg.svg', 'public') }}">
+                 src="{{ asset('assets/images/bolt_logo_loginpage--bg.svg', 'bolt') }}">
```

Comment:
@bobdenotter is this correct?

Reply:
Yes, this doesn't hurt. Currently `public` and `bolt` are the same, but a project _could_ overwrite `public`, potentially.

Reply:
As I remember I had to do that locally so the logo was showing up correctly.